addons/website/views/website_pages_views.xml

```diff
@@ -28,41 +28,53 @@
     </field>
 </record>
 
-<record id="website_page_properties_view_form" model="ir.ui.view">
-    <field name="name">website.page.properties.form.view</field>
-    <field name="model">website.page</field>
+<record id="website_page_properties_mixin_view_form" model="ir.ui.view">
+    <field name="name">website.page.properties.properties.form.view</field>
+    <field name="model">website.page.properties</field>
     <field name="arch" type="xml">
         <form js_class="page_properties_dialog_form">
             <group class="mt-4">
-                <field name="name" string="Page Title" placeholder="e.g. Home Page"/>
-                <field name="website_id" invisible="1"/>
-                <field name="url" string="Page URL" widget="page_url"/>
-
                 <label for="is_in_menu" string="In Menu"/>
                 <div class="o_row">
-                    <field name="is_in_menu" widget="boolean_toggle"/>
+                    <field name="is_in_menu" widget="boolean_toggle" options="{'autosave': False}"/>
```

Comment:
Indeed, and this is the case since 16.0. Should we backport this?

Btw, from 16.0 and up on runbots, the Show In Menu toggle works to remove the page from the menu, but not to add it again. If you agree I'll create a task if it doesn't already exist.

Reply:
We can probably backport it indeed (maybe move it in its own PR targeting stable directly), and you can create a task yes.

Reply:
I created the tasks, but this fix being important and given the lack of time I would opt to keep this in this PR. Without the fix, there are actually more bugs than before the PR that are caught by the tests, so this wouldn't be trivial to adapt.